### PR TITLE
Fix: typos in first half of ..\docs\scarpet\api

### PR DESCRIPTION
**Pull request by James103**
In the first half of ..\docs\scarpet\api, I have fixed many grammer, spelling, and consistency issues, as well as clarified some things that lacked explanation.

Let me know if there's anything that I can do to improve my contribution here.

### CODE REVIEW

File: docs/scarpet/api/BlocksAndWorldAccess.md [hunk 1]
```diff
@@ -267,51 +267,51 @@ poi(x,y,z,5) => [['nether_portal',0,[7,8,9]],['nether_portal',0,[7,9,9]]] // two
...
 
 ### `solid(pos)`
 
-Boolean function, true if the block is solid
+Boolean function, true if the block is solid.
 
 ### `air(pos)`
 
-Boolean function, true if a block is air.... or cave air... or void air.... or any other air they come up with.
+Boolean function, true if a block is air, cave air, or void air, or any other air they come up with.
```
Comment (gnembon) [inline docs/scarpet/api/BlocksAndWorldAccess.md:278]:
```suggestion
Boolean function, true if a block is air... or cave air... or void air... or any other air they come up with.
```

File: docs/scarpet/api/BlocksAndWorldAccess.md [hunk 2]
```diff
@@ -504,7 +504,7 @@ Deprecated as of scarpet 1.6, use `loaded_status(x) > 0`, or just `loaded(x)` wi
 
 ### `loaded_status(pos)`
 
-Returns loaded status as per new 1.14 chunk ticket system, 0 for inaccessible, 1 for border chunk, 2 for ticking, 
+Returns loaded status as per new 1.14 chunk ticket system, 0 for inaccessible, 1 for border chunk, 2 for (non-entity) ticking,
```
Comment (gnembon) [inline docs/scarpet/api/BlocksAndWorldAccess.md:507]:
```suggestion
Returns loaded status as per new 1.14 chunk ticket system, 0 for inaccessible, 1 for border chunk, 2 for redstone ticking, 
```

**Comment by James103** [comment]
Squash and merge (due to all the commits), or just merge?

File: docs/scarpet/api/BlocksAndWorldAccess.md [hunk 3]
```diff
@@ -539,10 +539,10 @@ Returns spawn potential at a location (1.16+ only)
 Checks wordgen eligibility for a structure in a given chunk. If no structure is given, or `null`, then it will check
  for all structures. If bounding box of the structures is also requested, it will compute size of potential
   structures. This function, unlike other in the `structure*` category is not using world data nor accesses chunks
-  making it preferred for scoping ungenerated terrain, but it takes some compute money to calculate the structure.
+  making it preferred for scoping ungenerated terrain, but it takes some compute power to calculate the structure.
```
Comment (gnembon) [inline docs/scarpet/api/BlocksAndWorldAccess.md:542]:
```suggestion
  making it preferred for scoping ungenerated terrain, but it takes some compute resources to calculate the structure.
```

**Comment by gnembon** [comment]
Hi - can you contact me on discord - I need to ask you one thing about carpet

**Comment by James103** [comment]
> I, James103, agree to licence fabric-carpet and my contributions to it under MIT licence.

Are there any other people who have contributed to the fabric-carpet project but haven't agreed to relicense their contributions under the MIT licence? Could you open an issue to keep track of this?

**Pull request merged**